x-pack/plugins/session_view/public/components/process_tree_node/styles.ts

```diff
@@ -30,14 +30,13 @@ export const useStyles = ({
   const cached = useMemo(() => {
     const { colors, border, size, font } = euiTheme;
 
+    const ALERT_INDICATOR_WIDTH = '3px';
```

Comment:
I thought we only use euiTheme variable (such as euiTheme.size.xs) instead of hardcoding value

Reply:
From code context I think we're discussing `red alert side line` :) We chose this width that is noticeable but not too intrusive. Especially in cases with many alerts one by one.

Reply:
thanks, @codearos, so I think we can see the tokens and the 4px baseline grid as a design standard or pattern to follow that provides guidance, but that's not written in the stone, some exceptions or adjustments can be made in favor of the User Experience

Reply:
@opauloh You are totally right!